code/modules/reagents/chemistry_reagents/medical.dm

```diff
@@ -269,7 +269,7 @@
 	properties = list(PROPERTY_NEOGENETIC = 3)
 
 /datum/reagent/medical/quickclot
-	name = "Quick Clot"
+	name = "wall-penetrating"
```

Comment:
```suggestion
	name = "Quick Clot"
```
I do not even know _how_.

Reply:
![image](https://user-images.githubusercontent.com/53100513/223785989-8d9d491b-62c9-4790-940d-949e6424a5e8.png)